rotkehlchen/api/rest.py

```diff
@@ -4993,6 +4993,22 @@ def add_tokens_to_spam(self, tokens: list[EvmToken]) -> Response:
                 )
                 AssetResolver.clean_memory_cache(token.identifier)
 
+                chain_balances = self.rotkehlchen.chains_aggregator.balances.get(
```

Comment:
this variable is not used elsewhere so you can just use directly in the for loop